src/status_im/constants.cljs

```diff
@@ -102,11 +102,7 @@
 
 (defn default-account-settings []
   {:web3-opt-in? true
-   :wallet       {:visible-tokens {:testnet #{:STT :HND}
-                                   :mainnet #{:SNT}
-                                   :rinkeby #{:MOKSHA :KDO}
-                                   :xdai    #{}
-                                   :poa     #{}}}})
+   :wallet       {:visible-tokens {}}})
```

Comment:
could you elaborate?

Reply:
@flexsurfer I don't understand why we used to force these default tokens on the user, when `visible-tokens` is empty the wallet autoconfigures and adds the tokens for which the user has a balance.